### DISCUSSION TITLE
Question - Connect to existing database

### DISCUSSION THREAD

**Discussion by breinbaas** [fastapi/full-stack-fastapi-template#1815] [Questions]
Hi all,

Short question, what would be the best direction to use an existing postgres database on another server? Or should I not bother using this nice template.

Regards,
Rob

**Comment by ieferrari** [comment]
maybe modifying in the content of  full-stack-fastapi-postgresql/backend/app/app/db/session.py  from 
`engine = create_engine(settings.SQLALCHEMY_DATABASE_URI, pool_pre_ping=True)`

to

`engine = create_engine("postgresql://"username:password@ip:port/db_name", pool_pre_ping=True)`
